group-generators/generators/index.ts

```diff
@@ -13,21 +15,30 @@ import SismoDomain from "./sismo-domains";
 import SismoGuest from "./sismo-guests";
 import SismoLensFollowers from "./sismo-lens-followers";
 import SismoMasqueradeLensFollowers from "./sismo-masquerade-lens-followers";
+import { DependencyContainer } from "tsyringe";
```

Comment:
I like when external dependencies are at the top. Maybe add a new linter rule?

Reply:
You are right, me too. I change it, and look to add a linter rule.

Reply:
fixed !